src/topology/bounded_continuous_function.lean

```diff
@@ -350,44 +360,85 @@ continuous functions from α to β inherits a normed group structure, by using
 pointwise operations and checking that they are compatible with the uniform distance. -/
 
 variables [topological_space α] [normed_group β]
-variables {f g : α →ᵇ β} {x : α} {C : ℝ}
+variables (f g : α →ᵇ β) {x : α} {C : ℝ}
 
-instance : has_zero (α →ᵇ β) := ⟨const 0⟩
+instance : has_zero (α →ᵇ β) := ⟨const α 0⟩
 
 @[simp] lemma coe_zero : (0 : α →ᵇ β) x = 0 := rfl
 
 instance : has_norm (α →ᵇ β) := ⟨λu, dist u 0⟩
 
 lemma norm_def : ∥f∥ = dist f 0 := rfl
 
+/- Formula for the norm. -/
```

Comment:
```suggestion
/-- Formula for the norm. -/
```

Reply:
Maybe some more description should be added to the docstring too.

Reply:
```suggestion
/-- The norm of a bounded continuous function is the supremum of `∥f x∥`.
We use `Inf` to ensure that the definition works if `α` has no elements. -/
```